recipe/meta.yaml

```
@@ -11,21 +11,21 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
     - sysroot_linux-64 2.17  # [linux64]
   host:
     - jupyter-packaging >=0.7.9,<0.8
```

Maybe try with the latest and greatest jupyter-packaging`?
```suggestion
    - jupyter-packaging >=0.12.3,<1
```